cep-????.md

```diff
@@ -0,0 +1,171 @@
+  | `*-armv7l`      | `armv7l`                         |
+  | `*-aarch64`     | `aarch64`                        |
+  | `*-arm64`       | `arm64`                          |
+  | `*-ppc64`       | `ppc64`                          |
+  | `*-ppc64le`     | `ppc64le`                        |
+  | `*-riscv64`     | `riscv64`                        |
+  | `*-s390x`       | `s390x`                          |
+  | `zos-z`         | `0`                              |
+  | Any other value | `0`                              |
+
+The build string MUST be overridable with the `CONDA_OVERRIDE_ARCHSPEC` environment variable, if set to a non-empty value.
+
+#### `__cuda`
+
+This virtual package MUST be present when the system exhibits GPU drivers compatible with the CUDA runtimes. When available, the version value MUST be set to the oldest CUDA version supported by the detected drivers (i.e. the formatted value of `libcuda.cuDriverGetVersion()`), constrained to the first two components (major and minor) and formatted as `{major}.{minor}`. The build string MUST be `0`.
+
+The version MUST be overridable with the `CONDA_OVERRIDE_CUDA` environment variable, if set to a non-empty value that can be parsed as a version string.
+
+#### `__glibc`
+
+This virtual package MUST NOT be present if the target platform is not `linux-*`.
+
+This virtual package MUST be present when the native and target platforms are both the same type of `linux-*` and GNU `libc` is installed in the system. The version value MUST be set to the system GNU `libc` version, constrained to the first two components (major and minor) formatted as `{major}.{minor}`. If the version cannot be estimated, the tool MUST set the version to a default value (e.g. `2.17`). 
+
+If the native platform does not match the target platform, the tool MAY export `__glibc` with its `version` field set to a default value (e.g. `2.17`) of its choice.  
+
+If the `CONDA_OVERRIDE_GLIBC` environment variable if set to a non-empty value that complies to the version string specification, the tool MUST export `__glibc` with its version value set to the value of the environemnt variable.
+
+The build string MUST always be `0`. 
+
+
+> The GNU `libc` version can be computed via:
+>
+> - Python's `os.confstr("CS_GNU_LIBC_VERSION")`
+> - `getconf GNU_LIBC_VERSION`
+> - `ldd --version`. Please verify that it references GNU `libc` or GLIBC. For non-standard installs, using a GLIBC compatibility layer, this may require locating the implementation and directly querying.
+
+#### `__linux`
+
+This virtual package MUST be present when the target platform is `linux-*`. Its version value MUST be set to the Linux kernel version, constrained to the first two to four numeric components formatted as `{major}.{minor}.{micro}.{patch}`. If the version cannot be estimated (e.g. because the native platform is not Linux), the tool MUST set `version` to a fallback value of its choice. The build string MUST be `0`.
```

Comment:
The Linux kernel upstream only defines `{major}.{minor}.{micro}`; anything beyond that (including the `{patch}` component you wrote) is part of the [distribution kernel's](https://www.kernel.org/releases.html#distribution-kernels) version string. I don't think conda tooling should expose those components since their semantics of those will vary from distribution to distributon; e.g., patch 42 on Fedora may differ from patch 42 on Ubuntu.

Reply:
Hm, this contradicts [this comment in `conda/conda`](https://github.com/conda/conda/blob/e74a2b9d8a74837afc3bcbef609fe4bd29572e16/conda/plugins/virtual_packages/linux.py#L20-L29). I'm happy to trim to three fields, but we'll need a source.